tests/test_tf_fixture.py

```diff
@@ -34,3 +36,95 @@ def test_tf_user_c(aws_sqs, aws_sns):
 
 def test_tf_user_many(aws_sqs, number_sequence):
     return number_sequence
+
+
+def test_tf_teardown_register():
+    fixture = tf.TerraformFixture(
+        tf_bin="fakebin",
+        plugin_cache="fakecache",
+        scope="function",
+        tf_root_module="fakeroot",
+        test_dir="fakedir",
+        replay=False,
+        teardown=tf.td.ON,
+    )
+
+    fixture.runner = MagicMock()
+    request = MagicMock()
+
+    fixture.create(request, MagicMock())
+
+    request.addfinalizer.assert_called()
+
+
+def test_tf_teardown_exception():
+    import subprocess
```

Comment:
any reason this import isn't top level?

Reply:
Not particularly, hold over from an older style of test writing from a past life. I can move it to top level